addon/components/denali-chip.hbs

```diff
@@ -0,0 +1,13 @@
+{{! Copyright 2020, Verizon Media. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<span
+  ...attributes
```

Comment:
I wasn't quite sure if attributes.class will be merged with the previously set class or replace it. Looking at the splat docs seemed to indicate it would be overwritten.

Reply:
Yeah, I could see how this isn't super obvious, it is at the bottom of this page: https://guides.emberjs.com/release/components/component-arguments-and-html-attributes/

>In addition, the class attribute is special, and will be merged with any existing classes on the element rather than overwriting them. This allows you to progressively add CSS classes to your components, and makes them more flexible overall.

Reply:
Oh, well there you go.